docs/tctl/admin/history_host/index.md

```diff
@@ -0,0 +1,25 @@
+---
+id: index
+title: tctl admin history_host
+description: Overview of history_host commands
+tags:
+  - reference
+  - tctl
+---
+
+::: NOTE
+
+The Temporal tctl documentation covers version 1.16 of the Temporal CLI. This information should not be definitely used for any beta or pre-production tctl versions.
```

Comment:
This is confusing.

Reply:
Same.
Is this to say that the information that pertains 1.16 is not applicable to other versions of tcl?

Reply:
```suggestion
This documentation reflects tctl version 1.16.
```
I would just leave it at that.

Reply:
However, we should probably publish this at the root level of this information... ?